### PR TITLE
feat(integrationDetailHistoryListViewItem)

### PR DESCRIPTION
**Pull request by kahboom**
IntegrationDetailHistoryListViewItem component + story, in preparation for IntegrationDetail page.
Relates to #58

### CODE REVIEW

**Comment by kahboom** [comment]
Not complete but let's just get this reviewed/merged in and I'll open a separate PR for the remainder of the Storybook work.

File: packages/ui/stories/Integration/IntegrationDetail.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+          i18nTextBtnPublish={text('', textBtnPublish)}
+          i18nTextHistoryMenuReplaceDraft={text(
+            '',
+            textHistoryMenuReplaceDraft
+          )}
+          i18nTextHistoryMenuUnpublish={text('', textHistoryMenuUnpublish)}
+          i18nTextLastPublished={text('', textLastPublished)}
+        />
+      </Grid.Col>
+    </Grid.Row>
```
Comment (riccardo-forina) [inline packages/ui/stories/Integration/IntegrationDetail.stories.tsx]: Can we refactor this - quite complicated - code in specialized components? We don't want to ask to the consumer of the IntegrationDetail to know this much of the inner bits of the UI library.

File: packages/ui/src/Integration/IntegrationDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  i18nTextHistory?: string;
+  i18nTextHistoryMenuReplaceDraft?: string;
+  i18nTextHistoryMenuUnpublish?: string;
+  i18nTextLastPublished?: string;
+  i18nTextNoDescription?: string;
+  i18nTextTabActivity: string;
+  i18nTextTabDetails: string;
+  i18nTextTableMetrics: string;
+  i18nTextTitle: string;
+  i18nTextVersion: string;
```
[the remark this answers is not in the document]
Comment (kahboom) [reply packages/ui/src/Integration/IntegrationDetail.tsx]: Sorry, yeah, not done yet with adding the tabs, was going to put that in a separate PR, but I can clean it up for this one.

File: packages/ui/src/Integration/IntegrationDetail.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          {this.props.children ? (
+            <ListView>{this.props.children}</ListView>
+          ) : (
+            <p>No History</p>
+          )}
+        </div>
+      </>
+    );
+  }
+}
```
[the remark this answers is not in the document]
Comment (kahboom) [reply packages/ui/src/Integration/IntegrationDetail.tsx]: In that case, should I extract the history and draft items into a separate component? Not referring to a route or anything, just a separate story and component. That way we'd keep this as strictly top level. In a way, it'd be like the IntegrationList and separating the items out into IntegrationListItem, etc.? This was kind of what I was trying to ask about how much we should be abstracting things.

File: packages/ui/stories/Integration/IntegrationDetail.stories.tsx (hunk 1 above)
Comment (kahboom) [reply packages/ui/stories/Integration/IntegrationDetail.stories.tsx]: Right, wasn't sure if all of the scenarios (published, unpublished, etc) should all be in the same story and component. I can break it up but just wasn't sure how much (e.g. per tab, also the history section).

**Comment by kahboom** [comment]
Just read your summarized feedback, which I think answers my question that we should definitely be having separate components for the history and horizontal view, but one single story, which kind of was my confusion to begin with. I'll fix this up now then.

**Pull request merged**